packages/hint-manifest-icons/src/meta.ts

```diff
@@ -0,0 +1,21 @@
+import { Category } from 'hint/dist/src/lib/enums/category';
+import { HintScope } from 'hint/dist/src/lib/enums/hint-scope';
+import { HintMetadata } from 'hint/dist/src/lib/types';
+
+const meta: HintMetadata = {
+    docs: {
+        category: Category.pwa,
+        description: `Description for manifest-icons`
```

Comment:
Don't forget to fill this out. Also, add a `name: "Manifest has icons"` property too (not required in general, but we add it to all the hints we ship as part of webhint).

Reply:
description and name added.